generating.cpp

```diff
@@ -38,7 +38,7 @@ Piece::Piece(char n, int o) {
  * In this case, 0 is the flag and 11 is the bomb
  */
 Piece::Piece(int v, int o) {
-	v > 10 ? value = 0 : value = v;
```

Comment:
Changed in a way that now a bomb actually can be made using an int value of 11. 
Value of the piece is set to 0 (which should be, because thats the value of a bomb) but the name is now still set to 'B'